packages/gatsby/package.json

```
@@ -0,0 +1,77 @@
+{
+  "name": "@sentry/gatsby",
+  "version": "5.16.1",
+  "description": "Offical Sentry SDK for Gatsby.js",
+  "repository": "git://github.com/getsentry/sentry-javascript.git",
+  "homepage": "https://github.com/getsentry/sentry-javascript/tree/master/packages/gatsby",
+  "author": "Sentry",
+  "license": "BSD-3-Clause",
+  "engines": {
+    "node": ">=6"
+  },
+  "main": "dist/index.js",
+  "module": "esm/index.js",
+  "types": "dist/index.d.ts",
+  "publishConfig": {
+    "access": "public"
+  },
+  "dependencies": {
+    "@sentry/browser": "5.16.1",
+    "@sentry/types": "5.16.1"
```

```suggestion
    "@sentry/apm": "5.16.1",
    "@sentry/react": "5.16.1",
    "@sentry/types": "5.16.1"
```

Missing reference to `@sentry/apm`

Also, shouldn't we be able to have `@sentry/react` as a base (vs. `@sentry/browser`)?

---

Yes `@sentry/react` can be used wherever `@sentry/browser` is used, the react package is a superset of the browser package (https://github.com/getsentry/sentry-javascript/blob/master/packages/react/src/index.ts#L1).

Also, we shouldn't need `@sentry/types` right? Right now the gatsby exported files don't use TS at all, so a user can just install `@sentry/types` manually if they want types.

---

It probably also makes sense then to makes docs changes + announcement of `@sentry/react` and `@sentry/gatsby` at the same time.

---

Yep, we can officially publish them at the same time.

---

sentry/react makes sense - though theres a possibility in the future it could make sense to inject sentry during compilation which is node.js. ks there actually anything I need to change in that case?

---

We can just `require("@sentry/node")` in `gatsby-node.js` right? Don't think it should change what we have right now.

---

:+1: yeah wasn't sure